airbyte-integrations/connectors/source-snapchat-marketing/source_snapchat_marketing/source.py

```diff
@@ -32,113 +118,49 @@
 #
 auxiliary_id_map = {}
 
-# The default value that is returned by stream_slices if there is no slice found: [None]
-default_stream_slices_return_value = [None]
-
 
 class SnapchatMarketingException(Exception):
     """Just for formatting the exception as SnapchatMarketing"""
 
 
-def get_depend_on_ids(depends_on_stream, depends_on_stream_config: Mapping, slice_key_name: str) -> List:
-    """This auxiliary function is to help retrieving the ids from another stream
+def get_parent_ids(parent) -> List:
```

Comment:
can we add a @functools.lru_cache on this method instead of supporting our own cache?